src/database/operations.py

```
@@ -0,0 +1,284 @@
+                   .filter(and_(
+                       CaseDocument.case_id == case_id,
+                       CaseDocument.document_type == document_type
+                   ))
+                   .order_by(desc(CaseDocument.created_at))
+                   .all())
+
+
+class ExtractionOperations:
+    """Database operations for extractions"""
+    
+    def __init__(self):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)